java/ql/src/utils/stub-generator/Stubs.qll

```diff
@@ -391,6 +402,24 @@ private string stubMember(Member m) {
   )
 }
 
+private string stubAnnotation(Annotation a) {
+  if exists(a.getAValue())
+  then result = a.toString() + "(" + concat(stubAnnotationValue(a.getAValue()), ",") + ")"
+  else result = a.toString()
+}
+
+private string stubAnnotiationSimpleValue(Expr value) {
+  result = value.(FieldAccess).getField().getQualifiedName() or
+  result = value.(Literal).toString()
+}
+
+private string stubAnnotationValue(Expr value) {
+  result = stubAnnotiationSimpleValue(value)
+  or
+  value instanceof ArrayInit and
+  result = "{" + concat(stubAnnotiationSimpleValue(value.(ArrayInit).getAnInit()), ",") + "}"
```

Comment:
Would it be necessary to order the elements of `concat` by their index in the array? Currently due to usage of `getAnInit()` it appears the values are ordered by their string representation.

Reply:
That's probably better, yes. Done in b18b4cf.